examples/api/agg_oo_sgskip.py

```diff
@@ -17,4 +17,4 @@
 ax.grid(True)
 ax.set_xlabel('time')
 ax.set_ylabel('volts')
-canvas.print_figure('test')
+fig.savefig('test')
```

Comment:
this line now makes it unclear as to why canvas was in the example, so maybe now that line should be cut?

Reply:
In order for it to work (without the pyplot interface), IIRC you need to create the `FigureCanvas` instance manually.

Reply:
Yup, you're right. I think that's even more reason to add a line/comment explaining that though since it's now sorta weirdly implicit since the canvas is created but not referenced.

Reply:
@story645 raises a good point, though: this is an *example*, so it should be helping the user to understand what is needed, and what is going on.  As it stands, it's baffling.  Is it necessary to retain the reference to the canvas?  If not, that variable should not be created, and a comment should be added to the effect that instantiating FigureCanvas adds a reference to the newly-created canvas to fig, which is all that is needed.

Reply:
Good point, edited accordingly.